avm/utilities/pipelines/sharedScripts/Set-ModuleReadMe.ps1

```diff
@@ -1609,13 +1609,15 @@ function Set-UsageExamplesSection {
             $bicepExample = ConvertTo-FormattedBicep @conversionInputObject
 
             # [6/6] Convert the Bicep format to a Bicep parameters file format
-            $bicepParamBlockArray = $bicepExample -split '\r?\n'
-            $topLevelParamIndent = ([regex]::Match($bicepParamBlockArray[0], '^(\s+).*')).Captures.Groups[1].Value.Length
-            $bicepParametersFileExample = $bicepParamBlockArray | ForEach-Object {
-                $line = $_
-                $line = $line -replace "^(\s{$topLevelParamIndent})([a-zA-Z]*)(:)(.*)", 'param $2 =$4' # Update any [    xyz: abc] to [param xyz = abc]
-                $line = $line -replace "^\s{$topLevelParamIndent}", '' # Update any [    xyz: abc] to [xyz: abc]
-                $line
+            if ($bicepExample.length -gt 0) {
```

Comment:
Hey @ReneHezser , so the Readme for which it failed was failing due to an empty parameter block in a test file?

Reply:
That is correct. $bicepExample was ""

Reply:
Might have been a good spot for a comment 😄 , but I'll resolve